Package.swift

```diff
@@ -54,7 +54,7 @@ let package = Package(
 
         // Messaging Push 
         .target(name: "CioMessagingPush",
-                dependencies: ["Common", "CioTracking"],
+                dependencies: ["CioTracking"],
```

Comment:
why did we remove it from here? was it never needed ?

Reply:
It was never needed. .

Because `CioTracking` depends on `Common` already, then any module that depends on `CioTracking` automatically include `Common` as a dependency. 

I removed it thinking that it would make the file easier to read since our team communicates like, *FCM depends on Tracking module*.